cmd/controller.go

```diff
@@ -58,6 +58,7 @@ func NewControllerCmd() *cobra.Command {
 			if err != nil {
 				return fmt.Errorf("error building kubeConfig: %w", err)
 			}
+			kubeCfg.Timeout = cfg.KubeClientTimeout
```

Comment:
Why is `cfg.KubeClientTimeout` not passed into `clientcmd.BuildConfigFromFlags`? That way we avoid people to forget assigning `kubeCfg.Timeout` (same question for [cmd/proxy.go line 55](https://github.com/hellofresh/kangal/pull/106/files#diff-c255784f7c4412e83a26aa04ec6156922b0a0bec2e2ff55e10f8981346da65a5R55))

Reply:
I don't understand, could you explain more? 🤔 

From what I understand, [`clientcmd.BuildConfigFromFlags` only takes path to `kubeconfig` file](https://github.com/kubernetes/kubernetes/blob/3433f099fb0b5c51d3b549372364f818242f7312/staging/src/k8s.io/client-go/tools/clientcmd/client_config.go#L613-L625), in which no timeout can be set to be used as the rest timeout config.

Reply:
A I didn't realize that `BuildConfigFromFlags` is part of k8s.
In that case why don't we create a function `BuildConfigFromFlags` within k8s where we require the `Timeout` as an argument? This would avoid all these `kubeCfg.Timeout =` all over the place :smile:

Reply:
Done in 68ee8a2